src/Culling/babylon.boundingSphere.ts

```diff
@@ -73,18 +82,31 @@ module BABYLON {
             const min = this.center.subtractToRef(tempRadiusVector, tmpVectors[1]);
             const max = this.center.addToRef(tempRadiusVector, tmpVectors[2]);
 
-            this.reConstruct(min, max);
+            this.reConstruct(min, max, this._worldMatrix, this._extraWorldExtent);

```

Comment:
scale should not clear `_worldMatrix` or `extraWorldExtent`. That's why we should keep them as members.